lib/transport/selenium-webdriver/httpclient.js

```diff
@@ -37,6 +37,16 @@ module.exports = function(settings, HttpResponse) {
 
     }
 
+    isDataRedacted(data) {
+      for (const value of Object.values(data)) {
+        if (typeof(value) === 'string' && value.includes('\uE000')) {
```

Comment:
We should match this with `Key.NULL`. Also, we should only check for this in case of a `POST` request being sent to an endpoint that ends with `/value`.

Reply:
updated in [0f6b9a7](https://github.com/nightwatchjs/nightwatch/pull/4066/commits/0f6b9a7e16ada3064d38a555bab1b6e85dffbe84)